tasks.py

```diff
@@ -517,67 +519,147 @@ def test_api(c, product, backend=''):
+
+godbc_config = (
+    "[pdns-mssql-docker]\n"
+    "Driver=FreeTDS\n"
+    "Trace=No\n"
+    "Server=127.0.0.1\n"
+    "Port=1433\n"
+    "Database=pdns\n"
+    "TDS_Version=7.1\n"
+    "\n"
+    "[pdns-mssql-docker-nodb]\n"
+    "Driver=FreeTDS\n"
+    "Trace=No\n"
+    "Server=127.0.0.1\n"
+    "Port=1433\n"
+    "TDS_Version=7.1\n"
+    "\n"
+    "[pdns-sqlite3-1]\n"
+    "Driver = SQLite3\n"
+    "Database = pdns.sqlite3\n"
+    "\n"
+    "[pdns-sqlite3-2]\n"
+    "Driver = SQLite3\n"
+    "Database = pdns.sqlite32\n"
 )
 
+def setup_godbc_mssql(c):
+    with open(os.path.expanduser("~/.odbc.ini"), "a") as f:
+        f.write(godbc_config)
+    c.sudo('sh -c \'echo "Threading=1" | cat /usr/share/tdsodbc/odbcinst.ini - | tee -a /etc/odbcinst.ini\'')
+    c.sudo('sed -i "s/libtdsodbc.so/\/usr\/lib\/x86_64-linux-gnu\/odbc\/libtdsodbc.so/g" /etc/odbcinst.ini')
+    c.run(f'echo "create database pdns" | isql -v pdns-mssql-docker-nodb {godbc_mssql_credentials["username"]} {godbc_mssql_credentials["password"]}')
+    # FIXME: Skip 8bit-txt-unescaped test
+    c.run('touch ${PWD}/regression-tests/tests/8bit-txt-unescaped/skip')
+
+def setup_godbc_sqlite3(c):
+    with open(os.path.expanduser("~/.odbc.ini"), "a") as f:
+        f.write(godbc_config)
+    c.sudo('sed -i "s/libsqlite3odbc.so/\/usr\/lib\/x86_64-linux-gnu\/odbc\/libsqlite3odbc.so/g" /etc/odbcinst.ini')
+
 @task
 def test_auth_backend(c, backend):
+    pdns_auth_env_vars = 'PDNS=/opt/pdns-auth/sbin/pdns_server PDNS2=/opt/pdns-auth/sbin/pdns_server SDIG=/opt/pdns-auth/bin/sdig NOTIFY=/opt/pdns-auth/bin/pdns_notify NSEC3DIG=/opt/pdns-auth/bin/nsec3dig SAXFR=/opt/pdns-auth/bin/saxfr ZONE2SQL=/opt/pdns-auth/bin/zone2sql ZONE2LDAP=/opt/pdns-auth/bin/zone2ldap ZONE2JSON=/opt/pdns-auth/bin/zone2json PDNSUTIL=/opt/pdns-auth/bin/pdnsutil PDNSCONTROL=/opt/pdns-auth/bin/pdns_control PDNSSERVER=/opt/pdns-auth/sbin/pdns_server SDIG=/opt/pdns-auth/bin/sdig GMYSQLHOST=127.0.0.1 GMYSQL2HOST=127.0.0.1 MYSQL_HOST="127.0.0.1" PGHOST="127.0.0.1" PGPORT="5432"'
+
     if backend == 'remote':
         ci_auth_install_remotebackend_test_deps(c)
 
     if backend == 'authpy':
         with c.cd('regression-tests.auth-py'):
-            c.run(f'PDNS=/opt/pdns-auth/sbin/pdns_server PDNS2=/opt/pdns-auth/sbin/pdns_server SDIG=/opt/pdns-auth/bin/sdig NOTIFY=/opt/pdns-auth/bin/pdns_notify NSEC3DIG=/opt/pdns-auth/bin/nsec3dig SAXFR=/opt/pdns-auth/bin/saxfr ZONE2SQL=/opt/pdns-auth/bin/zone2sql ZONE2LDAP=/opt/pdns-auth/bin/zone2ldap ZONE2JSON=/opt/pdns-auth/bin/zone2json PDNSUTIL=/opt/pdns-auth/bin/pdnsutil PDNSCONTROL=/opt/pdns-auth/bin/pdns_control PDNSSERVER=/opt/pdns-auth/sbin/pdns_server SDIG=/opt/pdns-auth/bin/sdig GMYSQLHOST=127.0.0.1 GMYSQL2HOST=127.0.0.1 MYSQL_HOST="127.0.0.1" PGHOST="127.0.0.1" PGPORT="5432" WITHKERBEROS=YES ./runtests')
+            c.run(f'{pdns_auth_env_vars} WITHKERBEROS=YES ./runtests')
+        return
+
+    # to avoid sudo cd error
```

Comment:
I'm not sure this comment makes sense here any more

Reply:
You are correct! I will clean that up now.